src/modal/alert-modal.component.ts

```diff
@@ -118,8 +118,8 @@ export class AlertModal extends BaseModal implements AfterViewInit {
 		this.closeModal();
 	}
 
-	dismissModal() {
-		if (this.onClose && this.onClose() === false) {
+	dismissModal(trigger) {
+		if (this.onClose && this.onClose(trigger) === false) {
 			return;
 		}
 		this.closeModal();
```

Comment:
Would
```
if (this.onClose && this.onClose(trigger)) {
	this.closeModal()
}
```
work?

Reply:
Should work so long as onClose is always defined

Reply:
Okay, and since we don't want onClose to always be defined it wouldn't 👍